doc/source/_static/js/tags.js

```diff
@@ -23,22 +23,26 @@ window.addEventListener('load', () => {
       .then(panelTagMap => {
 
           for (const panelId in panelTagMap) {
-              let element = document.getElementsByClassName(panelId)[0];
-
-              // For each panel, attach data tags to the 4-th parent of the panel,
-              // which is the "div" element that we can later toggle.
-              // Sphinx Panels is too inflexible to allow us to attach data tags
-              // directly to the container.
-              for (let i = 0; i < 4; i++) {
-                  if (element.parentNode) {
-                      element = element.parentElement;
-                      element.setAttribute('data-tags', panelTagMap[panelId]);
-                  }
-                  else {
-                      console.log(panelId + ' has no parent element,' +
-                          'please check if the panel has been tagged correctly.');
+              try { // panel might not be in _this_ gallery
+                  let element = document.getElementsByClassName(panelId)[0];
+
+                  // For each panel, attach data tags to the 4-th parent of the panel,
+                  // which is the "div" element that we can later toggle.
+                  // Sphinx Panels is too inflexible to allow us to attach data tags
+                  // directly to the container.
+                  for (let i = 0; i < 4; i++) {
+                      if (element.parentNode) {
+                          element = element.parentElement;
+                          element.setAttribute('data-tags', panelTagMap[panelId]);
+                      }
+                      else {
+                          console.log(panelId + ' has no parent element,' +
+                              'please check if the panel has been tagged correctly.');
+                      }
                   }
               }
+              // simply skip panels not in this gallery
+              catch(err) {}
```

Comment:
One thing I noticed is that we start with "all" being selected. If I select e.g. "PyTorch", then "all" gets deselected and "PyTorch" gets selected (Expected). However, when I unselect "PyTorch", "all" does not get selected again. All panels show up so this is more of a detail but maybe it's easy to fix